packages/e2e-test-utils-playwright/src/admin/visit-site-editor.ts

```diff
@@ -54,13 +54,13 @@ export async function visitSiteEditor(
 
 		// Wait for the canvas loader to appear first, so that the locator that
 		// waits for the hidden state doesn't resolve prematurely.
-		await canvasLoader.waitFor( { state: 'visible' } );
+		await canvasLoader.waitFor( { state: 'visible', timeout: 60000 } );
```

Comment:
I'm not sure this change will have any impact to be honest, The spinner is supposed to render right away, while all the other asynchronous stuff are being loaded. 

Did you actually notice improvements with this change or is it a guess.

Reply:
It's just a guess. But we have pretty consistent test failures due to elements on the page not being available after timeout period. So I do think it will help, and it can't hurt!

> The spinner is supposed to render right away,
We need to check elements on the page, after page has loaded, so I do think this will help

Reply:
I'm not convinced to be honest. For me, this change is really useless as we already have a timeout in the next line. Do you have any example of consistent test failure that we can debug further.

Reply:
This is where the test fails, waiting for the canvasLoader to be visible. With cache dissabled and a throttled network connection of Fast 4g in chrome, it takes close to 9s have the spinner visible on my local machine. So that's cutting it pretty fine. I think this allows a bit more leeway for congested or slow gh action containers.

Reply:
This is the flaky test that started this https://github.com/woocommerce/woocommerce/issues/52404

Reply:
Ok that's convincing enough for me. I'm happy to ship. 9s to see the spinner is a problem in itself though (even in 4G), you might want to check why is that as well.